.gitignore

```diff
@@ -26,7 +26,6 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 
-**/public/settings.json
-**/public/datagateway-dataview-settings.json
-**/public/datagateway-download-settings.json
-**/public/datagateway-search-settings.json
+**/public/*settings*.json
+!**/public/*settings.example.json
```

Comment:
Bit late unfortunately but I may have just stumbled across a tiny mistake here. I tried this in OperationsGateway's .gitignore and line 29 caused the example settings files to be ignored as well, thus ignoring line 30. I believe what would work is changing line 29 to:
```
**/public/*settings.json
```
This ensures actual settings files are ignored and files ending in settings.example.json are not.

Reply:
May not even need line 30 with the above fix...

Reply:
I figured out my problem - I called my file **settings-example.json, not **settings.example.json. Your code is just fine! Apologies for that